torch/csrc/jit/frontend/sugared_value.h

```diff
@@ -31,7 +31,11 @@ struct TORCH_API SugaredValue
   // what can we do with this thing?
   // use it as a value e.g.  `this + 4`
   virtual Value* asValue(const SourceRange& loc, Function& m) {
-    throw ErrorReport(loc) << kind() << " cannot be used as a value";
```

Comment:
Can you move this to `PythonValue` (which is the specific closed over value). https://github.com/pytorch/pytorch/blob/35a36c128032c59d0e69f5b32f8c4b81eddec673/torch/csrc/jit/python/python_sugared_value.h#L32

Reply:
Done.